src/Screens/Bookmark/BookmarkIcon.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from "react";
+import { View } from "react-native";
+import { useDispatch, useSelector } from "react-redux";
+import { addBookmark, removeBookmark } from "../../Redux/bookmarkSlice";
+import Icon from "react-native-vector-icons/FontAwesome";
+import styles, { Color } from "../../Shared/styles";
+import { RoutingItems } from "../../Data/RoutingLookups";
+
```

Comment:
Please have a quick description of this component

Reply:
Done.